src/javascript/app/pages/user/metatrader/metatrader.config.js

```diff
@@ -209,8 +224,6 @@ const MetaTraderConfig = (() => {
                             if (is_ok) resolve();
                             else resolveWithMessage();
                         });
-                    } else {
-                        resolve();
```

Comment:
why was this removed?

Reply:
@negarn Since we covered rest of the account types, isn't it checking for demo accounts, which we added so this would be a duplicate.